src/kps/Main.java

```diff
@@ -23,6 +24,9 @@ public static void main(String[] args) throws XMLException, FileNotFoundExceptio
         Simulation simulation;
         try {
             simulation = SimulationXML.readSimulationFromFile(new FileInputStream(args[0]));
+            for (Mail m : simulation.getMail()){
+                m.setCalculatedRoute(simulation.buildCalculatedRoute(m.getFrom(), m.getTo(), m.getPriority()));
+            }
```

Comment:
The original mail events don't have calculated routes. I might just modify the mail event to calculate the routes if it doesn't exist.